Source/Core/InputCommon/ControllerInterface/DualShockUDPClient/DualShockUDPProto.h

```diff
@@ -242,15 +242,16 @@ struct Message
   template <class ToMsgType>
   std::optional<ToMsgType> CheckAndCastTo()
   {
-    u32 crc32_in_header = m_message.header.crc32;
+    const u32 crc32_in_header = m_message.header.crc32;
     // zero out the crc32 in the packet once we got it since that's whats needed for calculation
     m_message.header.crc32 = 0;
-    u32 crc32_calculated = CRC32(&m_message, sizeof(ToMsgType));
+    const u32 crc32_calculated = CRC32(&m_message, sizeof(ToMsgType));
     if (crc32_in_header != crc32_calculated)
     {
-      NOTICE_LOG(SERIALINTERFACE,
-                 "DualShockUDPClient Received message with bad CRC in header: got %u, expected %u",
-                 crc32_in_header, crc32_calculated);
+      NOTICE_LOG_FMT(
+          SERIALINTERFACE,
+          "DualShockUDPClient Received message with bad CRC in header: got {}, expected {}",
```

Comment:
Just wondering, would those CRCs be better suited for output as 8-character hexadecimal? Since thats what a CRC is usually presented as in many other places (rather than the 32-bit integer that it is right now)